app/assets/javascripts/templates/search_tooltip.ejs

```diff
@@ -0,0 +1,34 @@
+<div class="col-sm-12">
```

Comment:
We're trying to avoid creating new EJS files, can we use React/JSX here instead?

Reply:
Actually, yeah. This one will take no time to swap to JSX. 👍